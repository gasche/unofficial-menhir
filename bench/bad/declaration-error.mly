%token FOO BAR <int> /* the error is the type */
%type <int> INT
%token BAZ::::: QUUX /* the error is the colon */
%token HOP

%%

main: HOP BAZ QUUX { () }

