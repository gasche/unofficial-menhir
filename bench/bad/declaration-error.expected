File "declaration-error.mly", line 1, characters 0-20:
Syntax error in a %token declaration.
Here are sample valid declarations:
  %token DOT SEMICOLON
  %token <string> LID UID
File "declaration-error.mly", line 3, characters 10-11:
Syntax error inside a declaration.
Did you perhaps forget the %% that separates declarations and rules?
